packages/@expo/prebuild-config/src/plugins/unversioned/expo-splash-screen/withIosSplashColors.ts

```diff
@@ -108,9 +108,11 @@ async function writeContentsJsonAsync(
   directory: string,
   { colors }: { colors: ContentsJsonColor[] }
 ): Promise<void> {
-  await fs.ensureDir(directory);
+  if (!fs.existsSync(directory)) {
+    await fs.promises.mkdir(directory, { recursive: true });
+  }
```

Comment:
```suggestion
  await fs.promises.mkdir(directory, { recursive: true });
```

nit: same ensureDir migration without check exists